app/models/manageiq/providers/vmware/cloud_manager/ovf_template.rb

```diff
@@ -98,6 +102,10 @@ def parse_vapp_networks(ovf)
     end
   end
 
+  def bool(el, xpath, default: false)
+    text(el, xpath) == 'true' ? true : default
+  end
```

Comment:
Does this really work as intended? If XML contains
```xml
<vcloud:Enabled>false</vcloud:Enabled>
```
and I call

```ruby
bool(..., default: true)
```

then I'll get `true` which is wrong. The default parameter should only be returned when XML contains no entry for the xpath. So I'd rather go with:

```ruby
def bool(el, xpath, default: false)
  (match = el.elements[xpath]) ? match.text.downcase == 'true' : default
end
```